examples/bri-3/src/bri/zeroKnowledgeProof/services/circuit/circuitService.interface.ts

```diff
@@ -7,6 +7,8 @@ export interface ICircuitService {
     pathToCircuit: string,
     pathToProvingKey: string,
     pathToVerificationKey: string,
+    pathToWitnessCalculator?: string,
```

Comment:
this seems to be optional here, but required in implementation

Reply:
This is again done to accommodate different zk protocols. `pathToWitnessCalculator` is not required if we want to use groth16 but need to be added when using plonk with snarkjs.